### PR TITLE
Add llvm-dwarfdump to package

### PR DESCRIPTION
**Pull request by agocke**
The runtime CI needs llvm-dwarfdump for NativeAOT, so add it to the Mono Tools package. Right now there doesn't seem to be a good reason to create a new package.

### CODE REVIEW

**Comment by akoeplinger** [comment]
https://github.com/dotnet/llvm-project/pull/287 is adding a `Microsoft.NETCore.Runtime.JIT.Tools` package, it'd fit better there. Should we wait until that PR is merged?

**Comment by agocke** [comment]
Didn't see that at all, yeah that sounds great, we'll wait for that.

**Comment by MichalStrehovsky** [comment]
> Didn't see that at all, yeah that sounds great, we'll wait for that.

The JIT team is also already publishing this package from the objwriter branch (https://github.com/dotnet/llvm-project/pull/256). The tool that is in that package (llvm-mca) is already used in the src/tests tree of the runtime repo, so it's _exactly_ what we'd want.

**Comment by agocke** [comment]
This has been updated to add llvm-dwarfdump to the JIT tools package

**Review by MichalStrehovsky — APPROVED** [review]
Looks good to me but @BrianBohe @markples @tihan should have a look as well - this tool is for objwriter testing after all.

**Comment by agocke** [comment]
Merging to unblock some testing. Happy to respond to any comments that come in later.

**Comment by BrianBohe** [comment]
Sorry for the late reply. I am okey with adding any tool you think would be necessary for testing or development. It shouldn't be a problem to build another piece of the same llvm platform

**Pull request merged**
